src/dvclive/live.py

```diff
@@ -123,6 +123,17 @@ def _init_cleanup(self):
         if self.dvc_file and os.path.exists(self.dvc_file):
             os.remove(self.dvc_file)
 
+    def _init_check_dvcyaml_overlap(self):
+        for stage in self._dvc_repo.index.stages:
```

Comment:
I will address in a follow-up: we should wrap any `dvc` operations to catch and not raise exception but log warning instead